### DISCUSSION TITLE
Flickering issue on android with swiper

### DISCUSSION THREAD

**Discussion by VISHWATEJA-G** [nolimits4web/swiper#5249] [Q&A]
When I am using swiper component [effect]="'cards'", the overall screen starts flickering when swiping the cards on only android device. 
When there are more than 20 cards, then only its starts to flicker other wise it works fine.
Any suggestions would be appreciated.

https://user-images.githubusercontent.com/94345292/144242193-9740dd9a-cd8c-48f6-8fe4-c767f1021f3c.mp4

**Comment by vltansky** [comment]
Is it Ionic? Which version are you using?
Can you make stackblitz / codesandbox example?
Can you try reproducing this with core swiper (without using swiper angular component)?

**Reply by VISHWATEJA-G** [reply]
Yes, this is IONIC, version latest.
Yes we tried with codesandbox example.
Here is the link https://codesandbox.io/s/wiwd6?file=/src/app/app.component.ts
We haven't tried with the core swiper.

**Reply by vltansky** [reply]
it's not flickering for me in the codesandbox you provided, is it for you?

**Reply by VISHWATEJA-G** [reply]
Are you tried with adding more than 25 cards?
Are you checked in android mobile?
